### ISSUE TITLE
H Y P E

### ISSUE THREAD

**Issue by ChromeUniverse** [stacksmashing/gb-poke-web#1]
I don't even know what this repo is about but I'm hyped anyway lmao.
I love watching Stacksmashing!

**Comment by ghost** [comment]
I hope it's about pokémon

**Comment by ChromeUniverse** [comment]
Yeah, probably something like an online version of Pokémon for the GB, with that RPi Pico Link adapter and some extra web client/server stuff
It's nice to see Stacksmashing experimenting with some web technologies for some of his projects

**Comment by ChromeUniverse** [comment]
Yeah probably something like an online version of Pokémon for the GB using the RPi Pico link adapter... It's nice to see Stacksmashing experimenting with some web technologies in his projects as well

edit: my fat finger slipped and closed the issue so I reopened it again :-P

**Comment by ghost** [comment]
lol
